app/models/task.py

```diff
@@ -1,5 +1,36 @@
 from app import db
+from datetime import datetime
 
 
 class Task(db.Model):
-    task_id = db.Column(db.Integer, primary_key=True)
+    task_id = db.Column(db.Integer, primary_key = True)
+    title = db.Column(db.String)
+    description = db.Column(db.String)
+    completed_at = db.Column(db.DateTime, nullable = True)
+    goal_id = db.Column(db.Integer, db.ForeignKey('goal.goal_id'), nullable=True)
+    goal = db.relationship("Goal", back_populates="tasks")
+
+    def to_dict(self):
+        if self.completed_at != None:
+            is_complete = True
+        else:
+            is_complete = False
+
+        return_dictionary = {
+            "id": self.task_id,
+            "title": self.title,
+            "description": self.description,
+            "is_complete": is_complete
```

Comment:
```suggestion
            "is_complete": True if completed_at else False
```
Using this syntax will allow you to conditionally set `is_complete` in a way that's a little bit cleaner than the if-else block you have.